File: src/main/resources/output/IFC4X3_python/Entities/IfcPhysicalQuantity/IfcPhysicalComplexQuantity/IfcPhysicalComplexQuantity.py
class IfcPhysicalComplexQuantity(IfcPhysicalQuantity):
    def __init__(self):
        self.HasQuantities: SET [1:?] OF IfcPhysicalQuantity = None
        self.Discrimination: IfcLabel = None
        self.Quality: IfcLabel = None
        self.Usage: IfcLabel = None

    # === WHERE CLAUSES ===
    # NoSelfReference : SIZEOF(QUERY(temp <* HasQuantities | SELF :=: temp)) = 0
    # UniqueQuantityNames : IfcUniqueQuantityNames(HasQuantities)

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcPhysicalQuantity/IfcPhysicalComplexQuantity/IfcPhysicalComplexQuantity.cs
public class IfcPhysicalComplexQuantity : IfcPhysicalQuantity
{
    public SET [1:?] OF IfcPhysicalQuantity HasQuantities { get; set; }
    public IfcLabel Discrimination { get; set; }
    public IfcLabel Quality { get; set; }
    public IfcLabel Usage { get; set; }

    // === WHERE CLAUSES ===
    // NoSelfReference : SIZEOF(QUERY(temp <* HasQuantities | SELF :=: temp)) = 0
    // UniqueQuantityNames : IfcUniqueQuantityNames(HasQuantities)
}
